app/assets/stylesheets/application.css.scss

```diff
@@ -16,9 +16,37 @@
  *= require_self
 
  */
+$neutral: #d9d9d9;
 
 @import "font-awesome";
 
 legend a {
   font-weight: lighter;
 }
+
+.side-column {
+  font-weight: lighter;
+  text-align: center;
+  ul {
+    margin: auto;
+    width: 70%;
+  };
+  li {
+    box-shadow: 0 1px 0 #ccc,
+               0 2px 0 #c9c9c9,
```

Comment:
Color literals like `#c9c9c9` should only be used in variable declarations; they should be referred to via variable everywhere else.